fbw-a32nx/src/systems/instruments/src/ISIS/LandingSystem.tsx

```diff
@@ -7,23 +7,28 @@ type DeviationIndicatorProps = {
     available: boolean
 }
 
-const DeviationIndicator: React.FC<DeviationIndicatorProps> = ({ deviation, available }) => (
-    <>
-        <rect x={0} y={0} width={200} height={20} fill="black" />
-        <circle className="StrokeWhite NoFill" cx={10} cy={10} r={7} />
-        <circle className="StrokeWhite NoFill" cx={55} cy={10} r={7} />
-        <line x1={100} x2={100} y1={2} y2={18} strokeWidth={4} stroke="white" />
-        <circle className="StrokeWhite NoFill" cx={145} cy={10} r={7} />
-        <circle className="StrokeWhite NoFill" cx={190} cy={10} r={7} />
-        { available
-            && (
-                <g transform={`translate(${(deviation * 50).toFixed(5)} 0)`}>
+const DeviationIndicator: React.FC<DeviationIndicatorProps> = ({ deviation, available }) => {
+    const maxDeviation = 1.778; // the maximum position the diamond can be while still fully inside scale
+
+    const dots = Math.sign(deviation) * Math.min(Math.abs(deviation / 0.4), maxDeviation);
```

Comment:
This is not dots if the max deviation is less than 2?

Reply:
not sure I understand. Max deviation is in dots. 1.778 dots to be exact.

I do agree that the variable isnt perfectly named so I am open for suggestions.

Reply:
The scale is +/- 2 dots. It doesn’t make it to the second dot?